cli.js

```diff
@@ -4,13 +4,18 @@ const logSymbols = require('log-symbols');
 const meow = require('meow');
 const isOnline = require('is-online');
 
-meow(`
+const cli = meow(`
 	Example
 	  $ is-online
 	  ${logSymbols.success} Online
```

Comment:
The flag needs to be documented. See how to do it here: https://github.com/sindresorhus/boxen-cli/blob/master/cli.js

Also need to update readme afterwards.